lib/oslc_dispatch.pl

```diff
@@ -14,6 +14,12 @@
 limitations under the License.
 */
 
+/** <module> OSLC Prolog request dispatch handlers.
+ *
+ * Use predicates in this module to register your HTTP handlers. The =IRISpec=
+ * shall follow the pattern of =|ns:resource|=, which translates into a URI
+ * pattern of =|/ns/resource|=. The handler with the highest priority wins.
```

Comment:
I think I actually missed the `once` + recursion in the `dispatch_to_handlers`, so I think you start from the highest prio handler and invoke them until one of them populates the `Context.graph_out`